slang.yaml

```
@@ -0,0 +1,5 @@
+base_locale: en
+input_directory: assets/i18n
+timestamp: false
+contexts:
+imports:
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)